cmd/virt-prerunner/dhcpv6.json.conf

```diff
@@ -0,0 +1,61 @@
+{
+"Dhcp6": {
```

Comment:
This only covers ipv6. Does this mean that we will still use dnsmasq for ipv4?

Reply:
✔

Reply:
Is it possible that we can use the same DNS server for both IPv4 and IPv6?

Reply:
Do you mean the same DHCP server here?

I tried `dnsmasq` server, it supports serving both DHCPv4 and DHCPv6, but the `shared-network` is not functional with IPv6, that means we have to assign an IPv6 from the pod subnet to bridge, it can only work in masquerade interface.

ISC has two DHCP servers, `kea-dhcp4` for DHCPv4, and `kea-dhcp6` for DHCPv6.

I haven't try ISC `dhcpd` server yet, not sure whether it can cover both DHCPv4 and DHCPv6.

Reply:
As ISC `dhcpd` is now [end-of-life](https://www.isc.org/blogs/isc-dhcp-eol/), ISC `kea-dhcpx` server is recommended.